src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+public enum Color {
+    Red,
+    Blue,
+    White,
+    Green
```

Comment:
Enum values should be named in uppercase letters. Please correct the naming according to the Java style guide.